### PR TITLE
Implemented #55 image upload

### PR DESCRIPTION
**Pull request by stanislav-sulima**

### CODE REVIEW

File: src/AppBundle/Controller/UploadController.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+            $em->persist($image);
+            $em->flush();
+
+            return new JsonResponse(array(
+                'image' => $image->getWebPath()
+            ));
+        }
+
+        return new JsonResponse('An error has occurred', 500);
```
Comment (holyspecter) [inline src/AppBundle/Controller/UploadController.php]: invalid form doesn't mean we have server error, 400 (Bad request) will be better here imo.

**Comment by holyspecter** [comment]
what about removing of an old file if an image was changed?

File: src/AppBundle/Controller/UploadController.php (hunk 1 above)
Comment (Ishugaev) [inline src/AppBundle/Controller/UploadController.php]: agree

Comment (stanislav-sulima) [inline src/AppBundle/Controller/UploadController.php]: Ok, I will fix it.

**Comment by stanislav-sulima** [comment]
@holyspecter, what do you mean?

**Pull request merged**